go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -187,8 +213,12 @@ func createPerNodePhysicalVIPs(isIPv6 bool, protocol v1.Protocol, sourcePort int
 			klog.Errorf("Failed to find node physical IPs, for gateway: %s, error: %v", gatewayRouter, err)
 			return err
 		}
+		// Get node-local endpoints only
+		if nodeLocal {
+			klog.V(5).Infof("ExternalTrafficPolicy=Local only getting local endpoints for node %s", util.GetWorkerFromGatewayRouter(gatewayRouter))
+			targetIPs = eps.LocalToNode(gatewayRouter).IPs()
```

Comment:
this function is used in this commit, but created in a later commit. The order of the commits should be fixed.